code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -534,6 +534,33 @@
 	..()
 	. = 1
 
+/datum/reagent/consumable/monkey_energy
+	name = "Monkey Energy"
+	description = "The only drink that will make you unleash the ape."
+	color = "#f39b03" // rgb: 243, 155, 3
+	taste_description = "barbecue and nostalgia"
+	glass_icon_state = "monkey_energy_glass"
+	glass_name = "glass of Monkey Energy"
+	glass_desc = "You can unleash the ape, but without the pop of the can?"
+
+/datum/reagent/consumable/monkey_energy/on_mob_life(mob/living/carbon/M)
+	M.Jitter(20)
+	M.dizziness +=1
+	M.drowsyness = 0
+	M.AdjustSleeping(-40, FALSE)
+	M.adjust_bodytemperature(-5 * TEMPERATURE_DAMAGE_COEFFICIENT, BODYTEMP_NORMAL)
+	..()
+
+/datum/reagent/consumable/monkey_energy/on_mob_metabolize(mob/living/L)
+	..()
+	if(ismonkey(L))
+		L.add_movespeed_modifier(type, update=TRUE, priority=100, multiplicative_slowdown=-0.75, blacklisted_movetypes=(FLYING|FLOATING))
+
+/datum/reagent/consumable/monkey_energy/on_mob_end_metabolize(mob/living/L)
+	if(ismonkey(L))
+		L.remove_movespeed_modifier(type)
```

Comment:
Remove the ismonkey(L) check here, or else monkeys will chug monkey energy, become human before the monkey energy leaves their system, and keep the movement speed modifier forever (even after the chem leaves their system).

Actually, humanization code is weird, so maybe it'll be fine. Still, better safe than sorry.